app/models/answer.rb

```diff
@@ -3,4 +3,9 @@ class Answer < ApplicationRecord
   belongs_to :author, class_name: 'User'
 
   validates :body, presence: true
+
+  def best_answer
```

Comment:
Usually, if the method in model save changes in db, uses '!'
And naming should clearly tell you what it is doing
For example this method, I thought that it will just return me the best answer, but it changes data)
So I think the better name for this method will be 'mask_best_answer!'